### ISSUE TITLE
[Feature Request] Drag and drop functionality

### ISSUE THREAD

**Issue by siddarththiru** [oliverschwendener/ueli#500]
G'day @oliverschwendener ,
I have recently downloaded your extremely powerful keystroke launcher UELI. I am missing one feature which I used very often on WOX and that is the drag and drop. Please can we have this implemented

**Comment by oliverschwendener** [comment]
What exactly do you want to drag&drop? I guess you mean dragging and dropping search results (files and folders) out of ueli into another application?

**Comment by walter6469** [comment]
Yes that is exactly what I mean

**Comment by walter6469** [comment]
> What exactly do you want to drag&drop? I guess you mean dragging and dropping search results (files and folders) out of ueli into another application?

Also I am not able to get the Everything Plugin working, I did read the article about it, but just typing 'es?' opens everything's console rather than givin us an option to search right there like WOX

**Comment by oliverschwendener** [comment]
https://github.com/oliverschwendener/ueli/issues/317

**Comment by walter6469** [comment]
> #317

Hey!!! #317 worked, I didn't extract the command line interface zip file to the Program files folder of Everything, would recommend you to make it a little clear in the app

**Comment by Stooovie** [comment]
Yes please, Listary is the only one on Windows that can do drag-n-drop. It´s tremendously useful for, say, adding footage and assets to a video editing app or Photoshop.

**Comment by turbodude** [comment]
+1 vote for the feature

**Comment by fabeit** [comment]
+1

**Comment by GovindKrishnan** [comment]
+1

**Comment by Warden20** [comment]
+1

**Comment by attaboyBrad** [comment]
Absolutely this.  I use Listary in parallel still just because I often need to drag and drop file attachments.  Even worse, right now when I forget and try to do that with Ueli, the arrows no longer work to navigate in Ueli results unless I hit ctrl+L.  Throws me off my focus every time.

**Comment by oliverschwendener** [comment]
The next release will support drag & drop for certain search result items.
